### ISSUE TITLE
How to convert correctly yolov10 from ultralytics sources?

### ISSUE THREAD

**Issue by ablyzniuk** [Qengineering/YoloV10-ncnn-Raspberry-Pi-4#3]
Please, can you give a precise, step by sted description how to convert ultralytics yolov10n model to the ncnn format. 
Params file you've got is different much from that that have been generated by ultralytics...

**Comment by Qengineering** [comment]
Indeed. YoloV10 is "very new". Some operations are not yet implemented in ncnn.
Until that time, adaptions need to be made in the onnx intermediate model as workaround.
[THU-MIG](https://github.com/THU-MIG), adapted the YoloV10 to onnx models. And [zhouweigogogo](https://github.com/zhouweigogogo), adapted the onnx model to the ncnn framework. 
Please consult them with your porting issues.

**Comment by ablyzniuk** [comment]
> Indeed. YoloV10 is "very new". Some operations are not yet implemented in ncnn.
> 
> Until that time, adaptions need to be made in the onnx intermediate model as workaround.
> 
> [THU-MIG](https://github.com/THU-MIG), adapted the YoloV10 to onnx models. And [zhouweigogogo](https://github.com/zhouweigogogo), adapted the onnx model to the ncnn framework. 
> 
> Please consult them with your porting issues.

Great thanks)))
